src/roomsyncroniser.ts

```diff
@@ -51,6 +51,11 @@ export class RoomSyncroniser {
 	) {
 		this.roomStore = this.bridge.roomStore;
 		this.mxidLock = new Lock(MXID_LOOKUP_LOCK_TIMEOUT);
+		if (this.config.metrics.enabled) {
+			const rooms = roomStore.getAll();
+			this.bridge.bridgeRoomMetric.set({type: "dm", protocol: this.bridge.protocol.id}, rooms.filter(room => room.isDirect).length))
+			this.bridge.bridgeRoomMetric.set({type: "group", protocol: this.bridge.protocol.id}, rooms.filter(room => room.isDirect).length))
```

Comment:
you meant to filter `!room.isDirect`? 😉

Reply:
ah, yes... i was missing coffee :)

Reply:
GitHub doesn't like force-pushes, it seems - changed in the newest version